firstname.lastname.txt

```diff
@@ -0,0 +1,5 @@
+Jonathan Krilosky
+I am okay with being addressed as Jon
+I am in section 294
+I prefer Java, but I am still learning Java as it is. I am open to learning new tech stack as well
+Something to know is I am still somewhat of a newbie
```

Comment:
Awesome! Thanks for the great responses.
Please make sure to clean up your repo to protect any private information from being public.